central/sensor/service/pipeline/complianceoperatorrules/pipeline.go

```diff
@@ -89,4 +137,32 @@ func (s *pipelineImpl) Run(_ context.Context, clusterID string, msg *central.Msg
 	}
 }
 
+func (s *pipelineImpl) processComplianceRuleV2(ctx context.Context, event *central.SensorEvent, clusterID string) error {
+	if !features.ComplianceEnhancements.Enabled() {
+		return errors.New("Next gen compliance is disabled.  Message unexpected.")
+	}
+
+	rule := event.GetComplianceOperatorRuleV2()
+
+	if val := rule.Annotations[v1alpha1.RuleIDAnnotationKey]; val == "" {
```

Comment:
@rhmdnd This validation was in the V1 rule ingest.  I had deleted it but I'm not sure why so I revived it in response to a comment.  The question for you, is this even possible or is this from some really old version of CO?